lib/ansible/modules/network/fortios/fortios_system_resource_limits.py

```diff
@@ -0,0 +1,395 @@
+                type: int
+            user:
+                description:
+                    - Maximum number of local users.
+                type: int
+            user_group:
+                description:
+                    - Maximum number of user groups.
+                type: int
+'''
+
+EXAMPLES = '''
+- hosts: localhost
+  vars:
+   host: "192.168.122.40"
+   username: "admin"
+   password: ""
+   vdom: "root"
+   ssl_verify: "False"
+  tasks:
+  - name: Configure resource limits.
+    fortios_system_resource_limits:
+      host:  "{{ host }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+      vdom:  "{{ vdom }}"
+      https: "False"
+      system_resource_limits:
+        custom_service: "3"
+        dialup_tunnel: "4"
+        firewall_address: "5"
+        firewall_addrgrp: "6"
+        firewall_policy: "7"
+        ipsec_phase1: "8"
+        ipsec_phase1_interface: "9"
+        ipsec_phase2: "10"
+        ipsec_phase2_interface: "11"
+        log_disk_quota: "12"
+        onetime_schedule: "13"
+        proxy: "14"
+        recurring_schedule: "15"
+        service_group: "16"
+        session: "17"
+        sslvpn: "18"
+        user: "19"
+        user_group: "20"
+'''
+
+RETURN = '''
+build:
+  description: Build number of the fortigate image
+  returned: always
+  type: str
+  sample: '1547'
+http_method:
+  description: Last method used to provision the content into FortiGate
+  returned: always
+  type: str
+  sample: 'PUT'
+http_status:
+  description: Last result given by FortiGate on last operation applied
+  returned: always
+  type: str
+  sample: "200"
+mkey:
+  description: Master key (id) used in the last call to FortiGate
+  returned: success
+  type: str
+  sample: "id"
+name:
+  description: Name of the table used to fulfill the request
+  returned: always
+  type: str
+  sample: "urlfilter"
+path:
+  description: Path of the table used to fulfill the request
+  returned: always
+  type: str
+  sample: "webfilter"
+revision:
+  description: Internal revision number
+  returned: always
+  type: str
+  sample: "17.0.2.10658"
+serial:
+  description: Serial number of the unit
+  returned: always
+  type: str
+  sample: "FGVMEVYYQT3AB5352"
+status:
+  description: Indication of the operation's result
+  returned: always
+  type: str
+  sample: "success"
+vdom:
+  description: Virtual domain used
+  returned: always
+  type: str
+  sample: "root"
+version:
+  description: Version of the FortiGate
+  returned: always
+  type: str
+  sample: "v5.6.3"
+
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.connection import Connection
+from ansible.module_utils.network.fortios.fortios import FortiOSHandler
+from ansible.module_utils.network.fortimanager.common import FAIL_SOCKET_MSG
+
+
+def login(data, fos):
+    host = data['host']
+    username = data['username']
+    password = data['password']
+    ssl_verify = data['ssl_verify']
+
+    fos.debug('on')
```

Comment:
it might be worth adding an option to configure debugging, unless it's explicitly needed by the module.

Reply:
That is an interesting proposal. Since it is an important change and we are close to the delivery date, can we agree on implementing it on the next round for 2.10?

Also take into account that we will deprecate fortiosapi as long as HTTPAPI is mandatory and the only possible way to connect.

Reply:
no worries, let's make it for 2.10, but I think that debug option would be nice ;)